src/main/java/marquez/service/JobService.java

```diff
@@ -129,13 +130,12 @@ public Job createOrUpdate(
         final NamespaceRow namespaceRow = namespaceDao.findBy(namespaceName.getValue()).get();
         final JobRow newJobRow = Mapper.toJobRow(namespaceRow.getUuid(), jobName, jobMeta);
         jobDao.insert(newJobRow);
+        jobs.labels(namespaceName.getValue(), jobMeta.getType().toString()).inc();
```

Comment:
what does this labels line do ?

Reply:
Labels are a Prometheus concept, see [labels](https://prometheus.io/docs/practices/naming/#labels). But, essentially, they add an extra dimension to metrics. Super helpful!